ERROR:__main__:2023-01-03 18:18:35,107:local variable 'portfolio_dict' referenced before assignment
Traceback (most recent call last):
  File "D:\Synced folder 2\tradex_opinionTrading\codes_2\main_strategy_run.py", line 128, in <module>
    strat_obj_dict[j].update()
  File "D:\Synced folder 2\tradex_opinionTrading\codes_2\strat7.py", line 763, in update
    self._set_qty()
  File "D:\Synced folder 2\tradex_opinionTrading\codes_2\strat7.py", line 394, in _set_qty
    self.my_lastbuy_price_yes, self.my_lastbuy_price_no = self.mybet.get_buy_price(self.event_id)
  File "D:\Synced folder 2\tradex_opinionTrading\codes_2\mybets.py", line 120, in get_buy_price
    df = self.get_event_holdings(id)
  File "D:\Synced folder 2\tradex_opinionTrading\codes_2\mybets.py", line 44, in get_event_holdings
    port_dict = self._get_portfolio_dict()
  File "D:\Synced folder 2\tradex_opinionTrading\codes_2\mybets.py", line 41, in _get_portfolio_dict
    return portfolio_dict
UnboundLocalError: local variable 'portfolio_dict' referenced before assignment
